### ISSUE TITLE
Airplay speaker devices not showing on 14.8 / 8.0.2

### ISSUE THREAD

**Issue by Stavrak2** [pwn20wndstuff/Undecimus#2282]
**Describe the bug**
When selecting the Airplay button in an app or on the Home Screen, the usual selection of a users AirPlay output speakers are not shown, or occasionally are shown but don't output audio when selected.

**To Reproduce**
Click on an AirPlay audio selection button in an app or in the Control Centre
Expect to see users list of local speaker devices (e.g. Sonos)
Usually only 'iPhone' is visible, or occasionally other speakers are shown but don't have an effect when selected.

**Device (please complete the following information):**
 - iOS Version: 14.8
 - iOS Device: iPhone 11 Pro Max
 - unc0ver Version: 8.0,2

**Place an "x" between the brackets if true:**
 - [X] this is a bug others will be able to reproduce
 - [X] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [X] this issue is present after a rootfs restore
 - [X] this issue is present on the latest version of unc0ver

**Comment by sween64** [comment]
Same here on iPhone XS

**Comment by ozzies-git** [comment]
Reproducilble on my iPhone Xs on 14.8 Unc0ver 6.0.2. After "Restore RootFS" it works fine again.
After reboot of the Airport Express the Airplay speakers are visible, however, once I disconnect from the Wifi network and reconnect they are gone. Sometimes shows up, but most of the time not visible.

**Comment by Stavrak2** [comment]
> Reproducilble on my iPhone Xs on 14.8 Unc0ver 6.0.2. After "Restore RootFS" it works fine again.
> After reboot of the Airport Express the Airplay speakers are visible, however, once I disconnect from the Wifi network and reconnect they are gone. Sometimes shows up, but most of the time not visible.

By restoring the RootFS do you not effectively remove the jailbreak?

You can also get it to work similarly by rebooting the phone and not jailbreaking.

**Comment by ozzies-git** [comment]
> 

Yes, rebooting and not jailbreaking works. However, I want to have a jailbroken phone **and** have Airplay work properly :D

**Comment by Stavrak2** [comment]
> Yes, rebooting and not jailbreaking works. However, I want to have a jailbroken phone **and** have Airplay work properly :D

Indeed but are you daring restoring RootFS fixed it permanently after jailbreaking also?

**Comment by ozzies-git** [comment]
> > Yes, rebooting and not jailbreaking works. However, I want to have a jailbroken phone **and** have Airplay work properly :D
> 
> Indeed but are you daring restoring RootFS fixed it permanently after jailbreaking also?

No sorry. Restoring RootFS fixed it as long as I do not re-jailbreak. As soon as I jailbreak the phone again, the problem returns.

**Comment by badger200** [comment]
This Fugu14 jailbreak is semi broken and sadly seems completely abandoned by @Linushenze. It renames the system daemon **analyticsd** which is why it breaks everyone's Battery Usage settings screens. It must hijack that daemon for the jailbreak presumably. It renames the original to **analyticsd.back** and executes it, but clearly there are iOS processes that fail due to this renaming. 

PS the jailbreak also frequently randomly freezes processes causing all sorts of issues, which can only be solved by manually grepping ps aux for " T. " and "kill -CONT pid" to resume them. This is the worst jailbreak I've ever used by far. Sadly it's the only game in town for my A12X/14.4 so it's still better than nothing.

**Comment by nikosntoykas** [comment]
Any fix for AirPlay not showing on iOS 14.8 jailbreak? 

Is our dev Pwn still alive? Can he do something with this stupid AirPlay problem?

Thanks in advance and best regards
